app/mailers/user_notifier/mailer.rb

```diff
@@ -21,6 +21,7 @@ def mail_attributes
       from: address_format(UserNotifier.system_email, @notification.from_name),
       reply_to: address_format(@notification.from_email, @notification.from_name),
       to: @notification.user.email,
+      cc: (JSON.parse @notification.metadata.gsub('=>', ':'))['cc'],#FIXME when upgrading to rails 4.2 which supports jsonb
```

Comment:
I thought about this, but if it was a standard field we would need to add it to all notification tables, even though for the time being it will only be used in one. Also, technically CC could be an array in which case JSON is more adequate.